source/dub/internal/vibecompat/inet/path.d

```diff
@@ -34,7 +34,7 @@ struct Path {
 	/// Constructs a Path object by parsing a path string.
 	this(string pathstr)
 	{
-		m_nodes = cast(immutable)splitPath(pathstr);
+		m_nodes = (splitPath(pathstr).idup);
```

Comment:
Is this a necessary part of the fix? Casting should be completely legal here and can in fact be dropped on recent compiler versions. I'd commit a change here to remove the cast for DMD >= 2.067.